EsperBenchmark/PassthroughBenchmark/src/main/java/Main.java

```diff
@@ -0,0 +1,205 @@
+import com.espertech.esper.common.client.*;
```

Comment:
Fix the issue of wildcard imports. These need to be replaced with single line imports.